src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseBuilder.java

```diff
@@ -0,0 +1,156 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.entities;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.AddressUk;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.IdValue;
+
+@Component
+public class AsylumCaseBuilder {
```

Comment:
I often write "Builder" pattern implementations so client code can chain the methods which for me at least makes it slightly easier to use and more fluent.  Matter of opinion I guess!

Reply:
Yeah let's do that, but as it is not a functional change can we leave it to another story -- keen to get this one over the line so I can rebase future work from this PR?

Reply:
Yes that's fine, I think getting this done is more important.